library/ssl_misc.h

```diff
@@ -65,6 +65,16 @@
 #define inline __inline
 #endif
 
+/* Legacy minor version numbers as defined by:
+ * - RFC 2246: ProtocolVersion version = { 3, 1 };     // TLS v1.0
+ * - RFC 4346: ProtocolVersion version = { 3, 2 };     // TLS v1.1
+ *
+ * We no longer support these versions, but some code still references those
+ * constants, for keep them for now until we clean up that code.
```

Comment:
comment is outdated. I don't think any additional cleanup is intended.

Reply:
Agreed, the comment should be changed to something like "We no longer support these versions, but some code still references those constants as part of negotiating with the peer, so keep them available internally."

Reply:
Done